src/content/en/tools/lighthouse/audits/tabindex.md

```diff
@@ -0,0 +1,32 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "No Element Has a Tabindex Attribute Greater Than Zero" Lighthouse audit.
+
+{# wf_updated_on: 2017-01-23 #}
+{# wf_published_on: 2017-01-23 #}
+
+# No Element Has a Tabindex Attribute Greater Than Zero  {: .page-title }
+
+## Why the audit is important {: #why }
+
+The `tabindex` attribute makes elements keyboard navigable.
+Positive values indicate an explicit navigation ordering of elements.
+Although this is valid, in practice it is extremely hard to do correctly and
+creates unusable experiences for users who rely on assistive technologies.
+
+See [Using tabindex](/web/fundamentals/accessibility/focus/using-tabindex)
+for more information.
+
+## How to pass the audit {: #how }
+
+<<_shared/query.md>>
+
+Set the `tabindex` of each of these elements to either `-1`, for elements
+that should not be keyboard navigable, or `0`, for elements that should.
```

Comment:
I'd suggest also adding, "if you need an element to appear earlier in the tab order, consider moving it earlier in the DOM."